src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -506,4 +508,14 @@ inline ngraph::float16 ie_abs(const ngraph::float16 &val) {
 
 OPENVINO_SUPPRESS_DEPRECATED_END
 
-}  // namespace CommonTestUtils
+}  // namespace utils
+}  // namespace test
+}  // namespace ov
+
+
+// openvino_contrib and vpu repo use CommonTestUtils::
+// so we need to add these names to CommonTestUtils namespace
+namespace CommonTestUtils {
```

Comment:
Please prepare PR to contrib and vpu repo to avoid duplications

Reply:
Ok. I will create it.